packages/sanity/src/tasks/src/tasks/components/activity/TasksActivityLog.tsx

```diff
@@ -134,39 +137,75 @@ type Activity =
 export function TasksActivityLog(props: TasksActivityLogProps) {
   const {value, onChange, path} = props
   const currentUser = useCurrentUser()
+  const {title: workspaceTitle, basePath} = useWorkspace()
 
   const {comments, mentionOptions, operation, getComment} = useComments()
   const loading = comments.loading
   const taskComments = comments.data.open
 
+  const handleGetNotificationValue = useCallback(
+    (message: CommentInputProps['value'], commentId: string) => {
+      const studioUrl = new URL(`${window.location.origin}${basePath}/`)
```

Comment:
Is it common for studio URLs to change? I guess it would only be stale for older tasks.